ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml

```diff
@@ -51,10 +51,12 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
+- as: functional
   commands: |
+    mkdir -p ../operator && cp -r . ../operator
```

Comment:
Why is this necessary for the dataplane operator? I ask because I don't think any other operators are using this, ie:
https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-main.yaml#L58-L63

Reply:
without it, it fails on permission:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/39475/rehearse-39475-pull-ci-openstack-k8s-operators-dataplane-operator-main-unit/1659191122259349504

https://github.com/openshift/release/pull/39368#discussion_r1195356672